modules/virt-configuring-certificate-rotation.adoc

```diff
@@ -0,0 +1,42 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-configuring-certificate-rotation_{context}"]
+
+Configure certificate rotation parameters to replace existing certificates. You can do this during OpenShift Virtualization installation in the web console or after installation in the YAML file.
+
+.Procedure
+
+. Open the `HyperConverged` Custom Resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Edit the `spec.certConfig` fields below. All values must be 10 minutes in length or greater to avoid overloading the system, and should be expressed as strings that comply with golang's ParseDuration format (https://golang.org/pkg/time/#ParseDuration):
```

Comment:
1. The ISG advises against using terms like "below" to describe the location of a thing. You can say something like "as shown in the following example" instead.
2. A few nits in the second sentence lead me to suggest this (optional) rewrite:
```
To avoid overloading the system, ensure that all values are greater than or equal to 10 minutes. Express all values as strings that comply with the link:https://golang.org/pkg/time/#ParseDuration[golang `ParseDuration` format].
```